docs/en/core-services/Catalog.md

```diff
@@ -102,17 +102,35 @@ $ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
     thrift://metastore_host:9083 default
 ```
 
-If you want specify config file for the UDB, please add `attachdb` option `-o catalog.db.config.file`.
-Each time you want to affect the config file, you can use `alluxio table sync`.
-
-If you want to write a configuration file for the UDB, please configure it like the following given example.
-It must contain an array `bypassSet`, the comma-separated table names will be bypassed.
-
-```json
-{
-  "bypassSet" : ["table1", "table2"]
-}
-```
+#### UDB Configuration File
+
+To specify a configuration file for the UDB, append an option `-o catalog.db.config.file` to
+`attachdb` command.
+Each time the configuration file is changed, you can use `alluxio table sync` to apply the changes.
+
+The configuration file is in JSON format, and can contain these configurations:
+
+1. Tables and partitions bypassing specification:
+
+    You can specify some tables and partitions to be bypassed from Alluxio, so that they will not be
+    cached in Alluxio, instead clients will be directed to access them directly from the UDB. 
+    This can be helpful when some tables and partitions are large, and accommodating them in the cache
+    is undesirable.
+
+    ```json
+    {
+      "bypass": {
+        "tables": [
+          "table1",
+          {"table": "table2", "partitions": ["table2_part1", "table2_part2"]}
+        ]
+      }
+    }
+    ```
+
+    You can specify which tables and partitions within these tables should be bypassed from Alluxio.
+    By specifying only the table name, all partitions of that table, if any, will be bypassed. Otherwise, 
+    you can specify specific partitions to bypass.
```

Comment:
can we explainthe above json file? e.g., are we by-passing entire table of `"table1"` and two partitions in `table2`? Am I correct? better to relate to our examples.

Reply:
Yes, you are correct. Clarified in 6664afc4.